pkg/driver/driver.go

```diff
@@ -12,6 +12,8 @@ type Driver interface {
 
 	CreateDisk() error
 
+	InspectDisk(diskName string) (*store.Disk, error)
```

Comment:
I don't think we need InspectDisk in driver.

As this is not doing anything driver specific. In the usage point we could directly call store.InspectDisk

Reply:
Makes sense, I just updated it in the latest revision